downstream/modules/platform/proc-operator-create_crs.adoc

```diff
@@ -19,3 +48,9 @@ This enables upgrading a specific deployment, like a controller, without updatin
 
 The recommended approach however, is to upgrade the operator and use the default image values. 
 ====
+
+.Verification
+Go to your {OperatorPlatformNameShort} deployment and click btn:[All instances] to verify if all instances deployed correctly.
+You should see the *{PlatformNameShort}* instance and the deployed *AutomationController*, *EDA*, and *AutomationHub* instances here.
+
+Alternatively you can check by the command line, run: `oc get route` 
```

Comment:
"Alternatively, you can verify whether all instances deployed correctly by running `oc get route` in the command line." This might be a clearer statement